Assets/Scripts/Actors/ActorPhysics.cs

```diff
@@ -58,7 +58,11 @@ public float sprintMoveSpeed
 		get { return _sprintMoveSpeed; }
 	}
 
-	float _jumpMoveSpeed = 0f;
+	[ReadOnly("Jump move speed")]
+	public float jumpMoveSpeed = 0f;
```

Comment:
This should be `[SerializeField]`, not public since we don't want other scripts writing to it.